CredentialProvider.Microsoft/CredentialProvider.Microsoft.csproj

```diff
@@ -65,11 +65,11 @@
 
   <ItemGroup>
     <FilesToSign Include="$(OutDir)/$(TargetFileName)">
-      <Authenticode>Microsoft</Authenticode>
+      <Authenticode>Microsoft400</Authenticode>
       <StrongName>StrongName</StrongName>
     </FilesToSign>
     <FilesToSign Include="$(IntermediateOutputPath)/$(TargetFileName)">
-      <Authenticode>Microsoft</Authenticode>
+      <Authenticode>Microsoft400</Authenticode>
```

Comment:
Is there some major difference between Microsoft and Microsoft400?

Reply:
Microsoft is SHA1, Microsoft400 is SHA2 :)

Reply:
This was something Zach did because signing wasn't working right somehow. I'm just incorporating his change from another branch here.

Reply:
SHA1 certs got deprecated ~4 months ago.

Reply:
OK cool, thanks!